friends/src/App.css

```diff
@@ -0,0 +1,60 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
Since you aren't usin the App-logo and the CSS that was initially installed with create-react-app you can get rid of it